### PR TITLE
feature: objects sorting

### PR DESCRIPTION
**Pull request by mohitpubnub**
Object Sorting feature implementation

### CODE REVIEW

File: .pubnub.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 --- 
 changelog:
+  - 
+    changes: 
+      - 
+        text: "Objects Sorting"
+        type: feature
+    date: April 10, 20  
+    version: v4.8.2
```
Comment (davidnub) [inline .pubnub.yml:9]: Missing the root property version increase `4.8.2`

**Comment by mohitpubnub** [comment]
> We should ensure the root `version` property is updated as well before deployment.

You mean there should be a property at root level like:

```
name: unity
version: "4.8.2"
changelog: 
  ......
```


But unity has separate file called VERSION which is updated to 4.8.2!
Still we need to add this to .pubnub.yml?

**Comment by crimsonred** [comment]
> > We should ensure the root `version` property is updated as well before deployment.
> 
> You mean there should be a property at root level like:
> 
> ```
> name: unity
> version: "4.8.2"
> changelog: 
>   ......
> ```
> 
> But unity has separate file called VERSION which is updated to 4.8.2!
> Still we need to add this to .pubnub.yml?

@mohitpubnub yes you need to the VERSION in .pubnub.yml. The version here overrides whats there in the VERSION file.

@davidnub thanks for catching this!

File: .pubnub.yml (hunk 1 above)
Comment (mohitpubnub) [reply .pubnub.yml:9]: Got this!

**Pull request merged**